FSHSchool/FSHOnline-Examples
# Node 18 + Manifest Tweak

This PR contains a few changes:
* Removes node_modules from change control
* Uses Node 18 by default
* Updates manifest script to not assume a specific folder name
* Updates manifest script to console log a summary of manifests

To test:
* `npm install` then `npm run build`
* Note the summary and note that only difference is timestamp
* Add some new examples and remove some old examples
* `npm run build` again
* Note the differences highlighted in the summary and confirm `index.json` differences in diff
